project/Dependencies.scala

```diff
@@ -49,11 +49,17 @@ object Dependencies {
   lazy val jjwtImpl = "io.jsonwebtoken" % "jjwt-impl" % Versions.jjwt % Runtime
   lazy val jjwtJackson = "io.jsonwebtoken" % "jjwt-jackson" % Versions.jjwt % Runtime
 
+  lazy val jsonParser = "com.google.code.gson" % "gson" % "2.10.1"
+
+  lazy val jwtDecoder = "org.springframework.security" % "spring-security-oauth2-jose" % Versions.spring
   lazy val nimbusJoseJwt = "com.nimbusds" % "nimbus-jose-jwt" % Versions.nimbusJoseJwt
+  lazy val bouncyCastle = "org.bouncycastle" % "bcprov-jdk15on" % "1.70"
 
   lazy val pureConfig = "com.github.pureconfig" %% "pureconfig" % Versions.pureConfig
   lazy val pureConfigYaml = "com.github.pureconfig" %% "pureconfig-yaml" % Versions.pureConfig
 
+  lazy val requests = "com.lihaoyi" %% "requests" % "0.8.0"
+
```

Comment:
```suggestion
```
I have the feeling that this is not used. Am I correct? Has this been used previously at some point?